napari/utils/colormaps/_accelerated_cmap.py

```diff
@@ -215,7 +215,7 @@ def _labels_raw_to_texture_direct_inner_loop(
 
 try:
     import numba
-except ModuleNotFoundError:
+except:
```

Comment:
```suggestion
except ImportError:
```

This will be enough to handle any problem with import